linkerd/admin/src/main/scala/io/buoyant/linkerd/admin/LinkerdAdmin.scala

```diff
@@ -57,7 +59,8 @@ class LinkerdAdmin(app: App, linker: Linker) {
     "/delegator" -> DelegateHandler.ui(linker),
     "/delegator.json" -> DelegateHandler.api(linker),
     "/routers.json" -> new RouterHandler(linker),
-    "/metrics" -> MetricsHandler
+    "/metrics" -> MetricsHandler,
+    "/config.json" -> new ConfigHandler(config, Linker.configInitializers)
```

Comment:
How would you feel about supporting "/config.yaml" as well? I'd accept "bad" as a valid answer :smile: